src/cata_utility.cpp

```diff
@@ -193,10 +194,16 @@ double logarithmic_range( int min, int max, int pos )
     const double LOGI_MIN = logarithmic( -LOGI_CUTOFF );
     const double LOGI_MAX = logarithmic( +LOGI_CUTOFF );
     const double LOGI_RANGE = LOGI_MAX - LOGI_MIN;
-    // Anything beyond [min,max] gets clamped.
-    if( pos < min ) {
+
+    if( min >= max ) {
+        debugmsg( "Invalid range" );
```

Comment:
That `debugmsg` could have more details, like: "logarithmic_range: minimum >= maximum ([value of minimum] >= [value of maximum])".

Also, requiring `min` to be strictly lesser than `max` (not allowing equality) could break something. Morale code uses it.